llvm/unittests/Transforms/Vectorize/VPlanTest.cpp

```diff
@@ -950,7 +950,7 @@ TEST_F(VPRecipeTest, CastVPWidenCallRecipeToVPUserAndVPDef) {
   Args.push_back(Op1);
   Args.push_back(Op2);
   Args.push_back(CalledFn);
-  VPWidenCallRecipe Recipe(Call, Fn, Args);
+  VPWidenCallRecipe Recipe(Call, Fn, Args, {});
```

Comment:
Is this change needed, here and below, given that the last DL argument defaults to {}?

Reply:
Removed, thanks